src/components/Navbar.js

```diff
@@ -357,16 +357,16 @@ class Navbar extends React.Component {
     saveTarget = () => {
         let bookNo = AutographaStore.bookId.toString();
         let that = this;
-        let translationContent = [];
+        //let translationContent = [];
```

Comment:
Please explain the changes in this file.

Reply:
In the above code when we editing the content, after auto save occurs after 3secs and was refreshing the currently editing data, hence the cursor position use to jump from the current editing position to the starting of the corresponding verse.
 Because of the above reason we have commented the above line of codes.